tests/unit/test_charm.py

```diff
@@ -31,21 +31,18 @@ def setUp(self):
             "app_name": self.harness.model.app.name,
         }
 
+    @patch_network_get(private_address="1.1.1.1")
```

Comment:
I'll review the unit tests in a separate review once the issues covered in `patroni.py` and `charm.py` are covered